src/pages/Main/BestItem.scss

```
@@ -0,0 +1,98 @@
+.best-seller-wrapper {
+  //width: 1100px;
+  // height: 545.8px;
+  margin: 150px auto 0px;
```

```suggestion
.best-seller-wrapper {
  margin: 150px auto 0px;
```
불필요해진 값들은 정리 부탁드립니다!

---

> 불필요해진 값들은 정리 부탁드립니다!

불필요값들 정리 리뷰 반영하였습니다!